src/OpenLoco/src/Vehicles/Routing.cpp

```diff
@@ -40,27 +40,12 @@ namespace OpenLoco::Vehicles
         }
     };
 
+    // Note: This is not binary identical to vanilla so cannot be hooked!
     struct LocationOfInterestHashMap
     {
-        static constexpr auto kMapSize = 0x400;
-        static constexpr auto kMapSizeMask = kMapSize - 1;
-        static constexpr auto kMaxEntries = 0x39C;
+        static constexpr auto kMinFreeSlots = 100; // There must always be at least 100 free slots otherwise the hashmap gets very inefficient
```

Comment:
If we are changing the size of it make this a prime number for better distribution properties. 97, 101, 103 if we want to keep it that small.

Reply:
I want to keep the logic the same for now so keeping this at 100.